recipes/napari-nikon-nd2/meta.yaml

```diff
@@ -0,0 +1,47 @@
+{% set name = "napari-nikon-nd2" %}
+{% set version = "0.1.3" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/napari-nikon-nd2-{{ version }}.tar.gz
+  sha256: 26f3a2a3d0cd3ad8756d84d036abea84ff30ecae2671ee7dd70f1a1c32d539a0
+
+build:
+  noarch: python
+  script: {{ PYTHON }} -m pip install . -vv
+  number: 0
+
+requirements:
+  host:
+    - nd2reader
+    - pip
+    - python >=3.6
+    - setuptools-scm
+  run:
+    - napari-plugin-engine >=0.1.4
+    - nd2reader
+    - numpy
+    - python >=3.6,<=3.9
```

Comment:
Another 3.10 incompatible package? Just checking @goanpeca

Reply:
Yep, correct :)

Reply:
For reference:

```
  File "/home/conda/staged-recipes/build_artifacts/napari-nikon-nd2_1643242176372/test_tmp/run_test.py", line 2, in <module>
    import napari_nikon_nd2
  File "/home/conda/staged-recipes/build_artifacts/napari-nikon-nd2_1643242176372/_test_env_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold/lib/python3.10/site-packages/napari_nikon_nd2/__init__.py", line 6, in <module>
    from ._reader import napari_get_reader
  File "/home/conda/staged-recipes/build_artifacts/napari-nikon-nd2_1643242176372/_test_env_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold/lib/python3.10/site-packages/napari_nikon_nd2/_reader.py", line 12, in <module>
    from nd2reader import ND2Reader
  File "/home/conda/staged-recipes/build_artifacts/napari-nikon-nd2_1643242176372/_test_env_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold/lib/python3.10/site-packages/nd2reader/__init__.py", line 2, in <module>
    from nd2reader.reader import ND2Reader
  File "/home/conda/staged-recipes/build_artifacts/napari-nikon-nd2_1643242176372/_test_env_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold/lib/python3.10/site-packages/nd2reader/reader.py", line 1, in <module>
    from pims import Frame
  File "/home/conda/staged-recipes/build_artifacts/napari-nikon-nd2_1643242176372/_test_env_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold/lib/python3.10/site-packages/pims/__init__.py", line 1, in <module>
    from pims.api import *
  File "/home/conda/staged-recipes/build_artifacts/napari-nikon-nd2_1643242176372/_test_env_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold/lib/python3.10/site-packages/pims/api.py", line 19, in <module>
    from .cine import Cine  # noqa
  File "/home/conda/staged-recipes/build_artifacts/napari-nikon-nd2_1643242176372/_test_env_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold/lib/python3.10/site-packages/pims/cine.py", line 29, in <module>
    from collections import Iterable
ImportError: cannot import name 'Iterable' from 'collections' (/home/conda/staged-recipes/build_artifacts/napari-nikon-nd2_1643242176372/_test_env_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold/lib/python3.10/collections/__init__.py)

```

Reply:
Awesome, thanks! 

@cwood1967, it would be awesome if you could fix that deprecated import in the next release of your plugin! That way we can ensure the napari ecosystem is 3.10 compatible. Of course no rush, just a suggestion! Thanks!